ecs-cli/modules/cli/compose/adapter/convert_test.go

```diff
@@ -663,3 +665,23 @@ func TestConvertCamelCaseToUnderScore(t *testing.T) {
 		})
 	}
 }
+
+func TestConvertToHealthCheck(t *testing.T) {
+	timeout := 10 * time.Second
+	interval := time.Minute
+	retries := uint64(3)
+	startPeriod := 2 * time.Minute
+	input := &types.HealthCheckConfig{
+		Test:        []string{"CMD", "echo 'echo is not a good health check test command'"},
+		Timeout:     &timeout,
+		Interval:    &interval,
+		Retries:     &retries,
+		StartPeriod: &startPeriod,
+	}
+	output := ConvertToHealthCheck(input)
+	assert.ElementsMatch(t, input.Test, aws.StringValueSlice(output.Command))
+	assert.Equal(t, int64(10), aws.Int64Value(output.Timeout))
```

Comment:
Nit: To make this less verbose, what do you think about switching the type casting here? I.e. `assert.Equal(t, aws.Int64(10), output.Timeout)`

Reply:
[aws.Int64](https://docs.aws.amazon.com/sdk-for-go/api/aws/#Int64) returns a pointer, so I'd be comparing the pointer values right? I don't see how the references would be guaranteed to be equal?

Reply:
As far as I understand it, `assert.Equal` compares the pointer values: https://github.com/stretchr/testify/blob/master/assert/assertions.go#L325

Reply:
Ah I see; that's cool!